--- runc_salsa/debian/rules.orig	2025-08-13 11:56:01.330364598 +0200
+++ runc_salsa/debian/rules	2025-08-13 11:56:34.407289094 +0200
@@ -13,6 +14,15 @@
 execute_after_dh_auto_build:
 	cd man && ./md2man-all.sh

+override_dh_auto_configure:
+	# needed by src/github.com/opencontainers/runc/vendor/google.golang.org/protobuf/internal/editiondefaults/defaults.go
+	mkdir -p _build/src/github.com/opencontainers/runc/vendor/google.golang.org/protobuf/internal/editiondefaults
+	cp vendor/google.golang.org/protobuf/internal/editiondefaults/editions_defaults.binpb _build/src/github.com/opencontainers/runc/vendor/google.golang.org/protobuf/internal/editiondefaults/
+	# needed by src/github.com/opencontainers/runc/main.go
+	mkdir -p _build/src/github.com/opencontainers/runc
+	cp VERSION _build/src/github.com/opencontainers/runc/
+	dh_auto_configure -O--buildsystem=golang -O--builddirectory=_build
+
 override_dh_auto_build:
 	dh_auto_build -- -tags "$(TAGS)" -ldflags "$(LDFLAGS)"
