--- runc_salsa/debian/patches/series.orig	2025-08-13 10:09:18.211136011 +0200
+++ runc_salsa/debian/patches/series	2025-08-13 10:09:29.323446143 +0200
@@ -1,13 +1,2 @@
-0001-skip-test-hugetlb_test.go.patch
-0002-skip-privileged-test-factory_linux_test.go.patch
 0003-skip-privileged-test-nsenter_test.go.patch
-0004-skip-test-cgroups_test.go.patch
 0005-skip-integration-when-no-dev-kmsg.patch
-0006-skip-test-paths_test.go.patch
-0007-skip-test-manager_test.go.patch
-0008-tests-enable-seccomp-default-action-tests-on-arm.patch
-0009-skip-test-file_test.go.patch
-0010-export-blockIODevice.patch
-0011-Compile-against-go-criu-v7.patch
-0012-TestDevicesSetAllow-Skip-test-when-cgroup-file-is-em.patch
-0013-loong64-support.patch
